tools/model_tools/src/openvino/model_zoo/_common.py

```diff
@@ -76,6 +76,7 @@
     'text_to_speech',
     'time_series',
     'token_recognition',
+    'background_matting',
```

Comment:
new task type also should be listed in https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/model_tools/README.md

Reply:
The list updated.